string taskID
string ref
---
uint8 status
string msg
